client/app/components/BaseContainer/StoryContainer/index.jsx

```diff
@@ -0,0 +1,22 @@
+// @flow
+import React from 'react';
+import { Story } from '../../Story';
+
+export type Props = {
+  data: any,
```

Comment:
I think we should be explicitly defining this prop type rather than using any as a fallback.